jwst/white_light/white_light.py

```diff
@@ -40,9 +40,20 @@ def white_light(input):
     log.debug("norders = %d, sporders = %s, ntables_order = %s",
               norders, str(sporders), str(ntables_order))
 
-    # Compute the flux sum for each integration in the input
+    # Create a wavelength mask, using cutoffs if specified, then
+    # compute the flux sum for each integration in the input.
+    low_cutoff = -1.
+    high_cutoff = 1.e10
+    if min_wave is not None:
+        low_cutoff = min_wave
+    if max_wave is not None:
+        high_cutoff = max_wave
+
     for i in range(ntables):
-        fluxsums.append(np.nansum(input.spec[i].spec_table['FLUX']))
+        wavelength_mask = np.where((input.spec[i].spec_table['WAVELENGTH']>=low_cutoff)&
```

Comment:
The built-in flake8 in my brain would like to see a white space before the `&`, but not a big deal.